src/api/payment-requests/payment-requests.md

```diff
@@ -881,15 +881,15 @@ Voiding a payment request will cancel the request and trigger any refunds if nec
 <a name="release">
 ### Release funds held for a pre authorization Payment Request **EXPERIMENTAL**
 
-When you call release on a pre auth payment request, any remaining funds that were being held for the authorization are returned to the asset. 
+When you call release on a preauth Payment Request there are two behaviors. In the event the preauth Payment Request was created and then release is called before the authorization process completes, the Payment Request is cancelled and a cancellation payment activity is returned. If the pre authorization has been completed, any remaining funds that were being held for the authorization are returned to the asset, and a release payment activity is returned. 
```

Comment:
Could we tease out the cancellation case to another paragraph? This first paragraph should be about the expected case, exceptions just make it harder to read.